.github/workflows/on_push_action.yml

```diff
@@ -5,14 +5,24 @@ on: [push,pull_request]
 jobs:
   ubuntu-build-and-test:
     runs-on: ubuntu-latest
-    container:
-      image: gnosispm/miplib-backends:1.1.0
     steps:
     - name: Checkout
       uses: actions/checkout@v2.0.0
-    - name: Build project
-      uses: ashutoshvarma/action-cmake-build@master
+    - name: Build Container
+      uses: docker/build-push-action@v2
       with:
-        build-dir: ${{ runner.workspace }}/build
-        build-type: Release
-        run-test: true
+        context: .
+        file: "${{ github.workspace }}/docker/Dockerfile"
+        push: false
```

Comment:
We should push this to GHCR IMO.

Reply:
Yeah. This is still on my radar. This ended up to be more of an overhaul of the deployment process than just updating `SCIP`.

Reply:
🙈